_sources/api/operations/collective-operations/group-call.rst

```diff
@@ -1,7 +1,7 @@
-Group Calls
+Group calls 
```

Comment:
is it expected?
![image](https://github.com/user-attachments/assets/75413b55-c602-496a-8eb4-d1cc5e62043f)